avif-coder/src/main/cpp/coder.cpp

```diff
@@ -47,7 +47,10 @@ jbyteArray encodeBitmap(JNIEnv *env, jobject thiz,
     }
     std::shared_ptr<heif_encoder> encoder(mEncoder,
                                           [](heif_encoder *he) { heif_encoder_release(he); });
-    heif_encoder_set_lossy_quality(encoder.get(), quality);
+    if (quality < 100) {
+        heif_encoder_set_lossy_quality(encoder.get(), quality);
```

Comment:
If you wish to include a quality parameter then an error have to be thrown when it's not clamped in range 0...100. Magic numbers or fixes will lead to misunderstanding and unclear errors